components/logins/android/src/main/java/mozilla/appservices/logins/DatabaseLoginsStorage.kt

```diff
@@ -144,33 +144,9 @@ class DatabaseLoginsStorage(dbPath: String) : AutoCloseable {
 fun migrateLoginsWithMetrics(newDbPath: String, newDbEncKey: String, sqlCipherDbPath: String, sqlCipherEncKey: String) {
     try {
         // last param is the "salt" which is only used on iOS.
-        val metrics = migrateLogins(newDbPath, newDbEncKey, sqlCipherDbPath, sqlCipherEncKey, null)
-        recordMigrationMetrics(metrics)
+        migrateLogins(newDbPath, newDbEncKey, sqlCipherDbPath, sqlCipherEncKey, null)
     } catch (e: LoginsStorageException) {
-        // leave all counters at zero, including duration, to hopefully
-        // make it a bit easier to identify total failure.
-        LoginsStoreMetrics.migrationErrors.add(e.toString())
-    }
-}
-
-/**
- * Records metrics for the sqlcipher -> sqlite migration.
- */
-fun recordMigrationMetrics(jsonString: String) {
-    try {
-        val metrics = JSONObject(jsonString)
-        LoginsStoreMetrics.migrationNumProcessed.add(metrics.getInt("num_processed"))
-        LoginsStoreMetrics.migrationNumSucceeded.add(metrics.getInt("num_succeeded"))
-        LoginsStoreMetrics.migrationNumFailed.add(metrics.getInt("num_failed"))
-        LoginsStoreMetrics.migrationTotalDuration.setRawNanos(metrics.getLong("total_duration") * 1_000_000)
-        val errors = metrics.getJSONArray("errors")
-        for (i in 0 until errors.length()) {
-            LoginsStoreMetrics.migrationErrors.add(errors.getString(i))
-        }
-    } catch (e: JSONException) {
-        // This shouldn't happen, but we don't want to crash.
-        // There's no logging configured here and it's not clear there's value.
-        // So meh
+        // Ignore all migration errors
```

Comment:
I believe the context is that it was decided that there's no point returning an error because there's literally nothing the consuming app can do about it, and we are never going to retry, so there's no point making them handle it.